code/datums/elements/climbable.dm

```diff
@@ -114,15 +114,13 @@
 ///Handles climbing onto the atom when you click-drag
 /datum/element/climbable/proc/mousedrop_receive(atom/climbed_thing, atom/movable/dropped_atom, mob/user, params)
 	SIGNAL_HANDLER
-	if(user == dropped_atom && isliving(dropped_atom))
-		var/mob/living/living_target = dropped_atom
-		if(isanimal(living_target))
-			var/mob/living/simple_animal/animal = dropped_atom
-			if (!animal.dextrous)
-				return
-		if(living_target.mobility_flags & MOBILITY_MOVE)
-			INVOKE_ASYNC(src, PROC_REF(climb_structure), climbed_thing, living_target, params)
-			return
+	if(user != dropped_atom || !isliving(dropped_atom))
+		return
+	if(!HAS_TRAIT(user, TRAIT_CAN_HOLD_ITEMS)) // If you can hold items you can probably climb a fence?
+		return
```

Comment:
IMO we should also add another trait that can bypass this for future use in case people want to make bears climb fences like

```suggestion
	if(!HAS_TRAIT(user, TRAIT_FENCE_CLIMBER) && !HAS_TRAIT(user, TRAIT_CAN_HOLD_ITEMS)) // you get a freebie on climbing fences if you have hands
		return
```

just so the behavior is a bit more atomic. also, shouldn't we be trait-checking `dropped_atom` instead of `user` since that's the actual thing that we move?

Reply:
well earlier on we return if they're not the same thing, but sure

Reply:
yeah i'd just rather it be consistent where we either do everything on `user` or everything on `dropped_atom`.